frontend/src/components/AppBarOnboarder.tsx

```diff
@@ -15,6 +15,7 @@ import {
 	KeyboardArrowDown as KeyboardArrowDownIcon,
 	Logout as LogoutIcon,
 } from "@mui/icons-material";
+import { useNavigate } from "react-router-dom"; // Import useNavigate for navigation
```

Comment:
We already had useNavigate on line 20. So should delete one of them, otherwise will cause error